README.md

```diff
@@ -1,2 +1,62 @@
-# ttax
-Tensor-Train toolbox in Jax
+﻿# TTAX: Tensor-Train toolbox on Jax
+Implementation of Tensor-Train toolbox in Jax containing several routines for working with tensors in TT format in Python.
+## Installation
+[JAX](https://github.com/google/jax/blob/master/README.md) and [Flax](https://github.com/google/flax#readme) are required for installation (see instructions there).
+Install TTAX from PyPi:
+```
+> pip install ttax
+```
+## Quick start
+This is a quick starting guide to look at the basics of working with ttax library. Our library provides routines for Tensor-Train object – a compact (factorized) representation of a tensor.
+
+In example below we create TT-tensor, multiply it by constant and convert it to full tensor format.
+```python
+!pip install ttax jax flax
```

Comment:
I think it's better not to add the installation into code snippets (i.e. I would just remove this line)